docs/hw/mechanical.md

```diff
@@ -1,7 +1,9 @@
 # iRobot® Create® 3 Mechanical System
-IMPORTANT: Always remove the Create® 3 robot’s faceplate prior to dismantling, adjusting, altering, or affecting the faceplate at the risk of damaging the robot.
```

Comment:
I feel like it's a little visually jarring to start this page with two warning boxes. Is there any explanatory text we could put before them (like on the electrical page)?

Reply:
Agree. I'll add some text there.

Reply:
I moved them down the page some too. Let me know if you think they should be moved somewhere else.

Reply:
I think this is great.